wallet/src/service/Offers.ts

```diff
@@ -14,6 +14,8 @@ import {
   OfferUIStatus,
   Offer,
 } from '../store/Offers';
+import { M } from '@agoric/vat-data';
+import { mustMatch } from '@agoric/store';
```

Comment:
available in Endo now
```suggestion
import { M, mustMatch } from '@endo/patterns';
```